recipes/so3/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+import os
+
+
+
+class So3Conan(ConanFile):
+    name = "so3"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/astro-informatics/so3"
+    description = "Fast and accurate Wigner transforms"
+    settings = "os", "arch", "compiler", "build_type"
+    topics = ("physics", "astrophysics", "radio interferometry")
+    options = {"fPIC": [True, False]}
```

Comment:
```suggestion
    options = {"fPIC": [True, False]}
```

fPIC options doesn't honor original CMake: https://github.com/astro-informatics/so3/blob/1.3.3rc1/CMakeLists.txt#L12

Reply:
I got rid of the homegrown option in the original cmake.